content/release-10-17-0/announcements-10-17-0.md

```diff
@@ -42,6 +42,26 @@ declaration for applications to start. Refer to the [Spring Security documentati
 
 #### Implemented
 
+##### Analytics Builder - Change of type for Clear Alarm input port
+
+In the [Alarm Output](https://documentation.softwareag.com/pab/10.17.0/en/webhelp/pab-webhelp/index.html#page/pab-webhelp%2Fre_AnaBui_block_reference_Output_CreateAlarm.html) block,
+the type of the **Clear Alarm** input port has changed from `boolean` to `pulse`.
+Thus any existing model which has a Boolean input to the port will now only trigger a clear alarm on the transition
+from `false` to `true`, instead of the entire time that the input remains `true`.
+If you wish to retain the old behavior, connect a
+[Pulse](https://documentation.softwareag.com/pab/10.17.0/en/webhelp/pab-webhelp/index.html#page/pab-webhelp%2Fre_AnaBui_block_reference_Flow_Manipulation_Pulse.html)
+block to the **Clear Alarm** input port to send a sequence of pulse signals into the **Alarm Output** block.
+However, it is unlikely that a model should be attempting to clear the same alarm multiple times,
+and the previous behavior had the effect of sending numerous unnecessary HTTP requests, potentially causing a drop in performance.
+For more information on the `pulse` type and how inputs are converted, see
+[The pulse type](https://documentation.softwareag.com/pab/10.17.0/en/webhelp/pab-webhelp/index.html#page/pab-webhelp%2Fco-AnaBui_pulse_type.html) and
+[Type conversions](https://documentation.softwareag.com/pab/10.17.0/en/webhelp/pab-webhelp/index.html#page/pab-webhelp%2Fco-AnaBui_type_conversions.html),
+both in the Analytics Builder documentation.
+
+##### Changes to microservice health endpoint
+
+For reasons of security and performance, the REST endpoint `/service/cep/health` no longer returns a comprehensive list of status values. All of the same information is still available from REST endpoints under `/service/cep/diagnostics/...`.
```

Comment:
can we move this to a separate PR?

There was an issue with this change and it got reverted, Richard will be making another attempt to get this in but it would be good to isolate this for now

Reply:
oh, I did not add this. I was only moving this to a different location.
Can we keep it for now? And then, when @rthrippleton cannot get it in, we can remove it on a separate PR?

Reply:
Whatever happens I'll get the health endpoint change in this release, even if I can't get it working for readiness (the problem we're having), so don't worry.